alma/alma.php

```diff
@@ -347,8 +347,8 @@ public function viewAccess()
             'AdminModules',
             true,
             [],
-            ['configure' => 'alma', 'module_name' => 'alma', 'tab_module' => 'payments_gateways']
-        ) . '&configure=alma&module_name=alma&tab_module=payments_gateways';
+            ['configure' => $this->name, 'module_name' => $this->name, 'tab_module' => $this->tab]
```

Comment:
this `getAdminLink` parameters seem's to be the same as given in `alma/controllers/admin/AdminAlmaConfig.php`

It is possible to factorise this part of code to avoid code duplication ? (DRY: Don't Repeat Yourself 😉 )

Reply:
I've refactor this 👍